src/demo/launcher-app/launcher-app.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="padding-15">
   <div>
     <div>
-      <f8launcher
+      <f8launcher flow="launch"
```

Comment:
The demo app shows two workflows; osio and launcher. Is there a third workflow where specifying 'launch' is necessary? Or, are we always hiding the pipelines?

If there are only ever going to be two workflows, then the flow input probably isn't necessary. That is, we could just omit the pipelines step if targetEnvironment is not 'osio'?

Reply:
see: https://github.com/fabric8-launcher/ngx-launcher/blob/master/src/demo/create-app/create-app.component.html#L5